src/partest/scala/tools/partest/nest/Runner.scala

```diff
@@ -526,7 +526,7 @@ class Runner(val testInfo: TestInfo, val suiteRunner: AbstractRunner) {
     // or, OK, we'll let you crash the compiler with a FatalError if you supply a check file
     def checked(r: CompileRound) = r.result match {
       case crash @ Crash(_, t, _) if !checkFile.canRead || !t.isInstanceOf[FatalError] => crash
-      case dnc => diffIsOk
+      case dnc @ _ => diffIsOk
```

Comment:
what does this change do?

Reply:
Convention that `dnc` is unused.

Reply:
... oh. I thought it meant "does not crash"

Reply:
"descriptive"

Reply:
`dnc` == `descriptively named code`